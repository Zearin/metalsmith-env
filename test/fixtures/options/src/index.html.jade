---
title: Environmental Variables
pretty: true
---
doctype html
html(lang="en")
  head
    title= title
  body
    h1= title
    p This is the environmental variable "npm_package_config_foo":
    code
      pre= npm_package_config_foo
