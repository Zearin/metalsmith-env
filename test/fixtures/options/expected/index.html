<!DOCTYPE html>
<html lang="en">
  <head>
    <title>Environmental Variables</title>
  </head>
  <body>
    <h1>Environmental Variables</h1>
    <p>This is the environmental variable "npm_package_config_foo":</p><code>
      <pre>bar overriden</pre></code>
  </body>
</html>